solvers/spams.py

```diff
@@ -0,0 +1,40 @@
+from benchopt import BaseSolver
+from benchopt import safe_import_context
+
+with safe_import_context() as import_ctx:
+    from spams import lasso, fistaFlat
+    import numpy as np
+    import scipy
+
+
+class Solver(BaseSolver):
+    name = 'spams'
+
+    install_cmd = 'conda'
+    requirements = ['numpy', 'mkl', 'pip:spams']
```

Comment:
you need to add `scipy` as a requirement here.

Reply:
done